groups/chips-rds/iam.tf

```diff
@@ -0,0 +1,48 @@
+data "aws_iam_policy_document" "rds_assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["rds.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_role_policy" "s3_integration" {
+  name        = "ipol-rds-${var.identifier}-policy"
+  role        = aws_iam_role.s3_integration.id
+
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "s3:GetObject",
+                "s3:PutObject",
+                "s3:ListBucket"
```

Comment:
Just wondered if you need DeleteObject?

Reply:
@sgapperch Good spot - added missing permission in [44a4bf1](https://github.com/companieshouse/chips-terraform/pull/164/commits/44a4bf1263f6cbc812d6549570efbb3123aa0716)